release notes/v0.47.0.md

```diff
@@ -0,0 +1,113 @@
+k6 `v0.47.0` is here 🎉! This release includes:
+
+## Breaking changes
```

Comment:
I agree with @heitortsergent on [the previous comment](https://github.com/grafana/k6/pull/3360#discussion_r1343132051) and want to extend it with the suggestion of the whole section renaming. 

```suggestion
## Deprecations
```

The thing is that none of the mentioned things doesn't break k6 right now, but we will do this in the future. But seems reasonable to reserve the `breaking changes` for the fundamental breaking changes to keep it urgency.